README.md

```diff
@@ -504,6 +504,37 @@ When this option is included in the command no other options are processed.
 nanoff --clearcache
 ```
 
+## Firmware archive
+
+By default, *nanoff* is designed to work with the latest version of the tool itself and of the firmware packages published in the online firmware repository. Command line arguments like
+
```

Comment:
This should be explaining how to use the fw archive. 
No need to add here explanations on the inner works of the tool or how is doing something or the reasoning.

Reply:
ok